definitions/tools/add_strelka_gt.cwl

```diff
@@ -55,6 +55,27 @@ requirements:
                             $id++;
                         }
 
+                        # parse GT, handling unparseable case with stderr
+                        my $parsed_n_gt = parse_gt($n_gt_str, \%ids);
+                        if ($n_gt_info eq "ref") {
+                            $n_gt = "0/0";
+                        } elsif (defined($parsed_n_gt)) {
+                            $n_gt = $parsed_n_gt;
+                        } else {
+                            my $id_keys = join(" ", sort keys(%ids));
+                            print STDERR "parse_gt for n_gt=\"$n_gt_str\" and ids=\"$id_keys\" resulted in undefined\n";
```

Comment:
Looks like other parts of this script are using `say` instead of `print` if you're interested in doing that here, too. I like the cleanliness of not needing to add `"\n"` manually with it.